README.md

```diff
@@ -1,4 +1,13 @@
 American-Gut
 ============
 
-American Gut open-access data and IPython notebooks
+American Gut open-access code and IPython notebooks
+
+A note about Data
+---
+
+American Gut sequences and metadata are deposited in [The European Bioinformatics Institute](http://www.ebi.ac.uk/ena) as ERP012803.
```

Comment:
`...are deposited in EBI as...` -> `...are deposited in EBI under the accession number...`
And make ERP012803 a link to the data: -> http://www.ebi.ac.uk/ena/data/view/ERP012803